### ISSUE TITLE
The error 'Keys for this account already exist' appears while restoring the account by a seed phrase

### ISSUE THREAD

**Issue by elina2015** [status-im/status-desktop#5529]
# Bug Report

## Steps to reproduce
1. Launch the app, click "I already use Status' - 'Enter a seed phrase'
2. Enter the seed phrase: radar pilot hello dirt actual receive cool recall asset else fault eye
3. Enter a Display Name, passwords, skip Biometry
#### Expected behavior
The onboarding is successfully finished
#### Actual behavior
1. The error 'Keys for this account already exist...' appears
2. 2. The error looks incorrect
<img width="891" alt="image" src="https://user-images.githubusercontent.com/14942081/163108589-4a369fa9-65ce-4d78-80f7-48f1c39d95cd.png">

### Additional Information

- Status desktop version: [commit 45f147fa956f6dbbfa64e3c6bb1ceb7f90626b9b](https://github.com/status-im/status-desktop/pull/5512)
- Operating System: macOS Big Sur, v. 11.6.6 (20G604)

**Comment by anastasiyaig** [comment]
might be fixed by https://github.com/status-im/status-desktop/pull/5520, can't reproduce it there, please take a look @elina2015

**Comment by jrainville** [comment]
Also, that message is maybe because you already imported that account before. You cannot re-import the same account twice. Like the message says, you'd need to delete that account first.

I added another issue to improve that behaviour so that the user can click a button that will delete the DB for them https://github.com/status-im/status-desktop/issues/5130

**Comment by saledjenic** [comment]
@elina2015 I tested this and I cannot reproduce the issue even I imported the same seed phrase in a loop.

see this:
![imported-accounts](https://user-images.githubusercontent.com/86303051/163415637-ef57cda7-9d18-45f0-9727-1494afd533ef.png)

Then I noticed different hash for the same seed phrases, what should be wrong, @osmaczko may confirm that.
Then I checked PK of imported accounts and saw that the keys are different.
So do we have an issue logged for that?
